### PR TITLE
Portable extinguishers/Breaching Charge whitelisted into toolbelt rig

### PR DESCRIPTION
**Pull request by P-MCT**
## About the PR

In CM13 Portable extinguishers/Breaching Charges can fit in the M276 Toolbelt rig.

Portable extinguishers can also fit in the combat toolbelt rig.

And Portable extinguishers/Breaching Charges can fit in the tools pouch.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- fix: Fixed portable fire extinguishers and breaching charges not fitting into the normal toolbelt rig and tools pouch.

### CODE REVIEW

**Comment by P-MCT** [comment]
Pardon I'm no coder and I've double checked the IDs, Both CMFireExtinguisherPortable and RMCExplosiveBreachingCharge should be viable under the whitelist for the toolbelts/pouch?

**Comment by QuietlyWhisper** [comment]
> Pardon I'm no coder and I've double checked the IDs, Both CMFireExtinguisherPortable and RMCExplosiveBreachingCharge should be viable under the whitelist for the toolbelts/pouch?

You're trying to add `RMCExplosiveBreachingCharge` as a tag to the whitelist which does not exist, you need to go to the file that contains `RMCExplosiveBreachingCharge` and make a new tag like so
image

File: Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Explosives/breaching_charge.yml [hunk 1]
```diff
@@ -1,3 +1,6 @@
+- type: Tag
+    id: RMCExplosiveBreachingCharge
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Explosives/breaching_charge.yml:2]: This won't get read properly if the indentation differs
```suggestion
- type: Tag
  id: RMCExplosiveBreachingCharge
```

**Review by DrSmugleaf — DISMISSED** [review]
Thanks for the PR!

**Pull request merged**